### PR TITLE
Feat/profile16 mail16 icons

### PR DESCRIPTION
**Pull request by kk-11**

### CODE REVIEW

File: src/Atoms/Icon/communication/mail/Mail16.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { IconBase } from '../../IconBase';
+import { IconProps } from '../../IconBase.types';
+
+const Mail16: React.FC<IconProps> = (props) => {
+  return (
+    <IconBase {...props} width={24} height={24}>
```
Comment (alvelid) [inline src/Atoms/Icon/communication/mail/Mail16.tsx:8]: 16

File: src/Atoms/Icon/general/profile/Profile16.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { IconBase } from '../../IconBase';
+import { IconProps } from '../../IconBase.types';
+
+const Profile16: React.FC<IconProps> = (props) => {
+  return (
+    <IconBase {...props} width={24} height={24}>
```
Comment (alvelid) [inline src/Atoms/Icon/general/profile/Profile16.tsx:8]: 16

Comment (jesperwiner) [inline src/Atoms/Icon/general/profile/Profile16.tsx:8]: need to be 16x16

File: src/Atoms/Icon/communication/mail/Mail16.tsx (hunk 1 above)
Comment (jesperwiner) [inline src/Atoms/Icon/communication/mail/Mail16.tsx:8]: here too,  16x16

**Review by jesperwiner — DISMISSED** [review]

**Review by alvelid — APPROVED** [review]

**Review by jesperwiner — APPROVED** [review]

**Review by mic1000 — APPROVED** [review]

**Pull request merged**